src/realm/util/interprocess_mutex.hpp

```diff
@@ -131,6 +132,11 @@ class InterprocessMutex {
     void free_lock_info();
 #else
     SharedPart* m_shared_part = nullptr;
+
+#ifdef _WIN32
+    HANDLE m_handle = 0;
```

Comment:
Windows API guarantees that no API method will ever return 0 as a valid HANDLE (in fact many methods return a NULL HANDLE to indicate error). So the correctness of the code is OK. I just find it more convencient that you can simply state `if(m_handle) ...` etc. And I think it's just as readible and easy to understand